### PR TITLE
No longer use "button" when referring to "Edit ID" menu item for triggers

### LINKED ISSUE #22705: Automations: Avoid the word "button" when referring to trigger menu
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

When setting up the "triggered by" condition in an automation or script the message

https://github.com/home-assistant/frontend/blob/a08c7a319fd8266f2214eb7c33d12c11e4ab2124/src/translations/en.json#L3274

comes up when no IDs have been set, yet.

### Describe the behavior you expected

The word "button" has resulted in some misleading translations here. It should avoid that by using

   No triggers have an ID set. You can edit IDs using each trigger's menu.

Note that the menu item this refers to is "Edit ID".

### Steps to reproduce the issue

n/a

### What version of Home Assistant Core has the issue?

core-2024.11.0

### What was the last working version of Home Assistant Core?

_No response_

### In which browser are you experiencing the issue with?

_No response_

### Which operating system are you using to run this browser?

_No response_

### State of relevant entities

_No response_

### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by NoRi2909**
The "triggered by" condition in an automation or script currently tells the user to "set an ID using the trigger menu button".

This change fixes this to refer to the "Edit ID" menu item that is meant here.

## Proposed change

Replace "You can set an ID using the trigger menu button."
with "You can edit IDs using each trigger's menu."

This also hints at the name of the menu item: "Edit ID"

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #22705 
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -3350,7 +3350,7 @@
                 },
                 "trigger": {
                   "label": "Triggered by",
-                  "no_triggers": "No triggers have an ID set. You can set an ID using the trigger menu button.",
+                  "no_triggers": "No triggers have an ID set. You can edit IDs using each trigger's menu.",
```
Comment (bramkragten) [inline src/translations/en.json:3353]: Do we have a user term for the 3 dot menu?

@googanhiem

**Review by bramkragten — DISMISSED** [review]

File: src/translations/en.json (hunk 1 above)
Comment (googanhiem) [reply src/translations/en.json:3353]: Checked with others, Docs call them a "three-dot menu".

`"No triggers have an ID set. Use the three-dot menu on the trigger and select Edit ID to assign one."`

Comment (NoRi2909) [reply src/translations/en.json]: OK, I've changed to the suggested string, only used "a trigger", not "the trigger" as it's up to the user to pick one or more to assign an ID at this point.

Comment (karwosts) [reply src/translations/en.json]: We do have a handful of references to "Overflow menu" in the current strings.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**